### ISSUE TITLE
Data architecture: determine how to include the additional reporting information for each activity

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-registration#1578]
#### Description:

Each activity will have special data requirements depending on WCI. Figure out a pattern allowing us to store that data

BA:
> The regs say that for each activity (although I believe SWRS asks for it by source type), any additional information referred to in the emission quantification methodology (WCI) must also be reported. For most activities types where there is a fuel, WCI specifies that the annual fuel consumption must be reported. There are certain ones (i.e., pneumatic venting or flaring at LFOs) where WCI doesn't require fuel to be reported. So, if there is a fuel associated with the activity, they should be reporting it. 

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking
